invenio_accounts/ext.py

```diff
@@ -205,6 +202,19 @@ def delay_security_email(msg):
         if app.config.get('ACCOUNTS_USERINFO_HEADERS'):
             request_finished.connect(set_session_info, app)
 
+        # Set Session KV store
+        if app.config.get('ACCOUNTS_SESSION_STORE_FACTORY'):
+            factory = load_obj_or_import_string(
+                app.config.get('ACCOUNTS_SESSION_STORE_FACTORY'))
+            session_kvstore = factory(app)
```

Comment:
Before calling the factory you can also have a check before if it is indeed a function, like: `if callable(factory): ...`